monitoring/oplog-populator/dashboard.py

```diff
@@ -0,0 +1,335 @@
+from grafanalib.core import (
+    ConstantInput,
+    DataSourceInput,
+    RowPanel,
+    Threshold,
+    Templating,
+    Template,
+)
+from grafanalib import formatunits as UNITS
+from scalgrafanalib import(
+    metrics,
+    layout,Target,
+    TimeSeries,
+    Dashboard,
+    Stat,
+    PieChart,
+)
+
+class Metrics:
+    RECONFIG_LAG = metrics.BucketMetric(
+        'oplog_populator_reconfiguration_lag_sec',
+        'job', 'connector', 'namespace'
+    ).with_defaults(
+        'job="${oplog_populator_job}"',
+        'connector=~"${connector}"',
+        'namespace="${namespace}"',
+    )
+
+    ACK_LAG = metrics.BucketMetric(
+        'oplog_populator_acknowledgement_lag_sec',
+        'opType', 'job', 'namespace'
+    ).with_defaults(
+        'job="${oplog_populator_job}"',
+        'namespace="${namespace}"',
+    )
+
+    CONNECTOR_CONFIG, BUCKETS, REQUEST_SIZE, PIPELINE_SIZE = [
+        metrics.CounterMetric(
+            name, 'job', 'namespace', *extraLabels
```

Comment:
`connector` is always present : should be move out of `extraLabels` and put here